src/components/widgets/accounts-live-balance/accounts-live-balance.directive.coffee

```diff
@@ -0,0 +1,90 @@
+#
+# Component generated by Impac! Widget Generator!
+#
+module = angular.module('impac.components.widgets.accounts-live-balance', [])
+module.controller('WidgetAccountsLiveBalanceCtrl', ($scope, $q, ChartFormatterSvc, $filter) ->
+
+  w = $scope.widget
+  w.isLiveBalance = true
+
+  # Define settings
+  # --------------------------------------
+  $scope.orgDeferred = $q.defer()
+  $scope.accountBackDeferred = $q.defer()
+
+  settingsPromises = [
+    $scope.orgDeferred.promise
+    $scope.accountBackDeferred.promise
+  ]
+
+  # Widget specific methods
+  # --------------------------------------
+
+  $scope.isDataFound=true
+  w.initContext = ->
+    $scope.isDataFound = w.content?
+
+  $scope.getName = ->
```

Comment:
do you need a method there? Can't you display `w.selectedAccount.name` directly in the view (I don't think this would raise an error even if none account is selected)

Reply:
Yep, 👍